README.md

```diff
@@ -116,6 +116,31 @@ async function sdkTest() {
 }
 ```
 
+#### Updates a template to Adobe App Builder Template Registry
+Updates a template to [Adobe App Builder Template Registry](https://github.com/adobe/aio-template-submission).
+
+**Note: It's only accessible with an IMS service token**
+
+```javascript
+const sdk = require('@adobe/aio-lib-templates');
+
+async function sdkTest() {
+    const templateRegistryClient = sdk.init({
+        'auth': {
+            'token': '<IMS_ACCESS_TOKEN>'
+        }
+    });
+    const templateId = '66431bb2f682a35bf4b1b002';
```

Comment:
Small nitpick else looks good, maybe we should write `const templateId = <Template_id>`

Reply:
yeah i concur with Arjun, I wouldn't be surprised if someone would just copy and paste this exact thing in 🤔